src/github.com/matrix-org/dendrite/federationapi/routing/events.go

```diff
@@ -45,11 +61,13 @@ func GetEvent(
 		&authResponse,
 	)
 	if err != nil {
-		return util.ErrorResponse(err)
+		resErr := util.ErrorResponse(err)
```

Comment:
These could also just be

```
return nil, &util.ErrorResponse(err)
```

Reply:
I couldn't believe it either. That is not allowed in go.